.circleci/config.yml

```diff
@@ -156,3 +198,13 @@ workflows:
           filters:
             tags:
               only: /.*/
+  nightly:
+    jobs:
+      - nightly
+    triggers:
+      - schedule:
+          cron: "15 * * * *"
+          filters:
+            branches:
+              only:
+                - nightly-ci
```

Comment:
This is only for testing the PR?

Reply:
yes, I will change this to `master` before merging, as well as the cron

Reply:
This would change to `master`, right?